### PR TITLE
Bug/194 update deploy workflow

### PR DESCRIPTION
**Pull request by KzuDemEvin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,8 +64,13 @@ It uses Spring boot and communicates directly with the mongoDb in a docker conta
 Run the docker container first, then the backend.
 
 ### Run Diveni with Docker and Docker Compose
+#### Run with pre-built docker images
```
Comment (stritti) [inline README.md:67]: Could you add a link to the default `docker-compose.yml` as reference?

Comment (stritti) [reply README.md:67]: I started documentation in doc-Branch: [docs/guide/install.md](https://github.com/Sybit-Education/Diveni/pull/198/files#diff-c749a8248e06dd9d3f555251b87a61d355c79c44f4df0123837be8da9838b7a4)

Comment (KzuDemEvin) [reply README.md:67]: > Could you add a link to the default `docker-compose.yml` as reference?

Done.



> Also https://github.com/Sybit-Education/Diveni/blob/main/docs/guide/setup.md should be updated. Probably in Readme just a link to `setup.md` to reduce redundancies.

The setup.md is deprecated, isn't it? It is also not mentioned in the navigation bar. In addition I think that setup.md and install.md should be merged. What do you think?

Comment (stritti) [reply README.md]: Yes, that is correct. could be merged.

Comment (KzuDemEvin) [reply README.md]: Done. `develop.md` contained already most of `setup.md`.
I just added the connectors to `install.md` and fixed the displaying of the nested list in `develop.md`.

**Pull request merged**